{"name": "test_add_member_address_8", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n1 Failed Assumptions:\n\ncase\\test_add_member_address.py:61: AssumptionFailure\n>>\tpytest.assume(result[\"body\"][\"code\"] != 200, \"断言开发自检字段\")\nAssertionError: 断言开发自检字段\nassert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               1 Failed Assumptions:\nE               \nE               case\\test_add_member_address.py:61: AssumptionFailure\nE               >>\tpytest.assume(result[\"body\"][\"code\"] != 200, \"断言开发自检字段\")\nE               AssertionError: 断言开发自检字段\nE               assert False\n\nd:\\python\\lib\\site-packages\\six.py:702: FailedAssumption"}, "description": "\n    添加地址,名字存在不和谐词\n    :return:\n    ", "start": 1615079423986, "stop": 1615079423985, "uuid": "3704b0c0-1d37-4bef-b44f-c4797f5d4e32", "historyId": "82310862186d5d911c7632f86c5bc3b3", "testCaseId": "5f1e4524fb31fdd3552b3f6c80759a07", "fullName": "case.test_add_member_address#test_add_member_address_8", "labels": [{"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_add_member_address"}, {"name": "host", "value": "DESKTOP-BE07TO9"}, {"name": "thread", "value": "9988-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_add_member_address"}]}